slick/src/main/scala/slick/basic/BasicBackend.scala

```diff
@@ -296,68 +296,85 @@ trait BasicBackend { self =>
     }
 
     /** Stream a part of the results of a `SynchronousDatabaseAction` on this database. */
-    protected[BasicBackend] def scheduleSynchronousStreaming(a: SynchronousDatabaseAction[_, _ <: NoStream, This, _ <: Effect], ctx: StreamingContext, continuation: Boolean)(initialState: a.StreamState): Unit = try {
-      ctx.getEC(synchronousExecutionContext).prepare.execute(new AsyncExecutor.PrioritizedRunnable {
-        private[this] def str(l: Long) = if(l != Long.MaxValue) l else if(GlobalConfig.unicodeDump) "\u221E" else "oo"
+    protected[BasicBackend] def scheduleSynchronousStreaming(a: SynchronousDatabaseAction[_, _ <: NoStream, This, _ <: Effect], ctx: StreamingContext, continuation: Boolean)(initialState: a.StreamState): Unit =
+      try {
```

Comment:
Most of the changes here are related to formatting. There is a lot going one in this method and we have up to 4 nested try/catch/finally blocks. 

I'm changing the formatting in a trial to make this code a little bit more readable. Some times the `try` is not aligned if the `catch` or `finally`. At the first line, we had a `try` on column 212 with the closing `catch` on column 6.

I'm also adding some break lines so the eyes don't need to go parse for each bracket in order to create logical unit in a human brain.